--- lirc-0.8.7/daemons/lircd.c.orig	2014-04-11 07:28:01.355612722 +0200
+++ lirc-0.8.7/daemons/lircd.c	2014-04-11 07:30:54.335191111 +0200
@@ -2091,7 +2091,12 @@
 			memset(&event, 0, sizeof(event));
 			event.type = EV_KEY;
 			event.code = input_code;
+			/* the input subsystem is handling repeats on its own, so no need
+			 * to send a repeat event here
+			 * this would only break the rate and delay settings
 			event.value = release ? 0 : (reps>0 ? 2:1);
+			 */
+			event.value = release ? 0 : 1;
 			if(write(uinputfd, &event, sizeof(event)) != sizeof(event))
 			{
 				logprintf(LOG_ERR, "writing to uinput failed");
