--- ntp-4.2.6p5/ntpdc/ntpdc_ops.c.orig	2014-04-15 15:43:33.426714023 +0200
+++ ntp-4.2.6p5/ntpdc/ntpdc_ops.c	2014-04-15 15:48:12.724244075 +0200
@@ -2959,7 +2959,7 @@
 		tscale = 1e-9;
 #endif
 	(void)fprintf(fp, "pll offset:           %g s\n",
-	    (int32)ntohl(ik->offset) * tscale);
+	    (u_long)ntohl(ik->offset) * tscale);
 	(void)fprintf(fp, "pll frequency:        %s ppm\n",
 	    fptoa((s_fp)ntohl(ik->freq), 3));
 	(void)fprintf(fp, "maximum error:        %g s\n",
